features/base/tracks/actionTypes.js

```diff
@@ -1,3 +1,4 @@
 export const TRACK_ADDED = 'TRACK_ADDED';
+export const TRACK_MUTE_CHANGED = 'TRACK_MUTE_CHANGED';
```

Comment:
@lyubomir exactly! I mean, one should document an action function, not an action name. And actually @legastero does that in https://github.com/legastero/jitsi-meet-react/blob/c8c203926cee7d9cfde12e8112d71aec9c2a02d9/features/base/tracks/actions.js#L145